src/coreclr/jit/forwardsub.cpp

```diff
@@ -221,7 +221,7 @@ class ForwardSubVisitor final : public GenTreeVisitor<ForwardSubVisitor>
                 // fgGetStubAddrArg cannot handle complex trees (it calls gtClone)
                 //
                 bool isCallTarget = false;
-                if (parent->IsCall())
+                if ((parent != nullptr) && parent->IsCall())
```

Comment:
`m_parentNode` also has some uses that seem to need fixing.

Reply:
Indeed. Should be addressed now.